rdt-open-reader/src/testCovidIndonesia/java/io/ona/rdt/util/CovidRDTJsonFormUtilsTest.java

```diff
@@ -845,6 +848,14 @@ else if (CovidConstants.FormFields.SENDER_NAME.equals(field.getString(JsonFormCo
             assertEquals(field.getString(VALUE), ANM_PREFERRED_NAME);
             numOfPopulatedFields++;
         }
+        else if (CovidConstants.FormFields.NAME_OF_HEALTH_FACILITY.equals(field.getString(JsonFormConstants.KEY))) {
```

Comment:
Could you also add a separate test to verify `CovidConstants.FormFields.FACILITY_NAME` and `CovidConstants.FormFields.HEALTH_FACILITY_NAME` fields are pre-populated? You can create a `JSONObject` with these fields or read the forms directly from assets and verify that the pre-population logic works as expected.

Reply:
fixed.